### PR TITLE
NagendraT_Assignment_36

### PR DESCRIPTION
**Pull request by NagendraTOct2020**

### CODE REVIEW

**Review by harshvegada — CHANGES_REQUESTED** [review]
Logic looks bit complex, 
Code works fine but check below code once

https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/1190/files

**Comment by NagendraTOct2020** [comment]
I have updated the assignment 36 and reduced the complexity of code 


Type a message

File: src/nagendra/array/UnionOfArray.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+public class UnionOfArray {
+
+	double[] getUnionOfArray(double[] arr1, double[] arr2) {
+
+		double[] input = new double[arr1.length + arr2.length];
+		System.arraycopy(arr1, 0, input, 0, arr1.length);
+		System.arraycopy(arr2, 0, input, arr1.length, arr2.length);
+		double[] output = new double[input.length];
+		int i = -000;
```
Comment (harshvegada) [inline src/nagendra/array/UnionOfArray.java:19]: this will consider as 0 only -000 make no sense

**Review by harshvegada — APPROVED** [review]
Code looks Good to me
Good to merge

**Pull request merged**